site/docs/master/oracle-config.md

```diff
@@ -0,0 +1,243 @@
+# Use Oracle Cloud as a Backup Storage Provider for Velero
+
+## Introduction
+
+[Velero](https://heptio.github.io/velero/master/) is a tool used to backup and migrate Kubernetes applications. We support [Oracle Cloud Object Storage](https://docs.cloud.oracle.com/iaas/Content/Object/Concepts/objectstorageoverview.htm) as a destination for Velero backups. 
```

Comment:
Unfortunately, we can't support and test storage providers beyond the ones we already do. The `We support` language needs to be changed.